store/BasalTestingSlice.js

```diff
@@ -0,0 +1,80 @@
+import { createSlice, createAsyncThunk } from '@reduxjs/toolkit';
+import axios from 'axios';
+import AsyncStorage from '@react-native-async-storage/async-storage';
+
+export const addBasalTest = createAsyncThunk(
+  'addTest',
+  async ({ numTest, glucose,date }) => {
+    try {
+      const jwt = await AsyncStorage.getItem('token');
```

Comment:
Nice, if you have time you should demo how asyncstorage works. Good job implementing it in your app